benchmarks/src/main/kotlin/chat/BenchmarkConfiguration.kt

```diff
@@ -0,0 +1,192 @@
+package chat
+
+import kotlinx.coroutines.CoroutineDispatcher
+import kotlinx.coroutines.asCoroutineDispatcher
+import kotlinx.coroutines.channels.Channel
+import kotlinx.coroutines.scheduling.ExperimentalCoroutineDispatcher
+import org.apache.commons.math3.stat.descriptive.moment.StandardDeviation
+import java.util.concurrent.ForkJoinPool
+
+// Benchmarks configuration to monitor
+
+/**
+ * Underlying thread pool size for coroutines.
+ */
+val THREADS = listOf(4, 8, 16, 1)
+/**
+ * Chat users count.
+ */
+val USER_COUNT = listOf(10000)
+/**
+ * Maximum percentage of all chat users a user can be friends with.
+ */
+val MAX_FRIENDS_PERCENTAGE = listOf(0.2)
+/**
+ * Coroutines channels type.
+ */
+val CHANNEL_TYPES = listOf(ChannelType.UNLIMITED, ChannelType.RENDEZVOUS, ChannelType.BUFFERED_1, ChannelType.BUFFERED_16, ChannelType.BUFFERED_256)
```

Comment:
All channels from `ChannelType`?

Reply:
Yes, right now this list contains all the channels from ChannelType. But as I explained in other comment, sometimes it is more convenient to set configuration explicitly.